docs/developers/charts.md

```diff
@@ -34,10 +34,10 @@ Dataset controllers must implement the following interface.
     draw: function(ease) {},
 
     // Remove hover styling from the given element
-    removeHoverStyle: function(element) {},
+    removeHoverStyle: function(element, datasetIndex, index) {},
```

Comment:
It looks like bubble, doughnut, polarArea and radar still do not accept these new parameters.

Reply:
I can't add them because the linter would complain about them being unused, but they're still passed and if anyone overrides the methods they'll still receive them